### ISSUE TITLE
Cant hear my own digipeats

### ISSUE THREAD

**Issue by al0apsk31** [ge0rg/aprsdroid#363]
Hello,

I have a digipeater at my home station. It consists of a 2M Transceiver, PC and Direwolf modem. I can use this to hear my own digipeats with a Kenwood d74. This works fine. 

I try to use APRSdroid, a Mobilinkd TNC3 and a Retevis RA89 Txcvr to send a position report. Position reports are received by Direwolf, it is then digipeated by the home station but not decoded in APRSdroid. When I use the APRSDroid/Mobilnkd/RA89 on the main USA network (144.390) all of my own position reports are received, digipeated and decoded by APRS droid just fine. 

To summarize...the APRS Droid setup works on the 144.390 side of things but not on the home digipeater. Again, the home digipeater behaves perfectly with a Kenwood d74. So I'm at a loss. I hope I explained this well enough to be understood.

Thanks for Reading. Greg, AL0A

**Comment by al0apsk31** [comment]
Upon further trouble shooting, I've found that the issue is related to  the T/R switching recovery time on the particular radio. The Radio does not return to full receive fast enough to hear the its own digipeated position report packet. Thus, the packet is cut off. My Kenwood style T/R switching is fast. The Retevis has a discernible delay.

**Comment by al0apsk31** [comment]
Upon further trouble shooting, I've found that the issue is related to the T/R switching recovery time on the particular radio. The Radio does not return to full receive fast enough to hear the its own digipeated position report packet. Thus, the packet is cut off. My Kenwood style T/R switching is fast. The Retevis has a discernible delay.
